subprojects/docs/src/snippets/tutorial/configureObject/kotlin/build.gradle.kts

```diff
@@ -1,12 +1,12 @@
-import java.text.FieldPosition
+class UserInfo(var name: String? = null, var email: String? = null)
```

Comment:
Better 👍 I'd put each parameter on a separate line so it looks more like the Groovy version.

Reply:
I felt this style was more idiomatic for data-only classes in Kotlin, like in [data classes](https://kotlinlang.org/docs/data-classes.html) - also, I tend to favor same line for comma-separated lists, new lines for semicolon separated ones. 

I will fold easily if you really prefer separate lines.

Reply:
Yes, please. It's not a hard rule but we tend to favor putting Kotlin field parameters in multiple lines (e.g, [`ModelKey`](https://github.com/gradle/gradle/blob/c20e024c5f8d707fd7c18620f2b99f0dc9f1127f/subprojects/configuration-cache/src/main/kotlin/org/gradle/configurationcache/cacheentry/ModelKey.kt#L23-L26)).

Reply:
Ah, but then you are letting Gradle's internal style guide (which is not really relevant to Gradle users) trump [Kotlin's own style guide](https://kotlinlang.org/docs/coding-conventions.html#class-headers). Seems questionable.

Reply:
Quoting the style guide:

> Classes with longer headers should be formatted so that each primary constructor parameter is in a separate line with indentation

Reply:
Hehe, I saw that, didn't consider this long (2 args), but I fold!